### ISSUE TITLE
webstorm无法定位自动引入的组件

### ISSUE THREAD

**Issue by enjoy-wind** [unplugin/unplugin-auto-import#318]
### Describe the bug

集成此插件，webstorm无法追溯定位引入的组件.
<img width="444" alt="image" src="https://user-images.githubusercontent.com/39360554/217210875-d76d64b5-ae37-4fa0-b823-ccf7e71b80d8.png">

### Reproduction

无

### System Info

```Shell
mac
```

### Used Package Manager

npm

### Validations

- [X] Follow our [Code of Conduct](https://github.com/antfu/.github/blob/main/CODE_OF_CONDUCT.md)
- [X] Read the [Contributing Guide](https://github.com/antfu/contribute).
- [X] Check that there isn't already an issue that reports the same bug to avoid creating a duplicate.
- [X] Check that this is a concrete bug. For Q&A, please open a GitHub Discussion instead.
- [X] The provided reproduction is a [minimal reproducible](https://stackoverflow.com/help/minimal-reproducible-example) of the bug.

**Comment by enjoy-wind** [comment]
升级webstorm版本吗
@userquin

**Comment by userquin** [comment]
No, I just file a new issue to include support in JetBrains IDE, for example rigth now we can run Vitest tests, a few months ago there wasn't support.

**Comment by enjoy-wind** [comment]
另外好像还有个问题，引入标签时，他好像还会自动import

**Comment by userquin** [comment]
> 另外好像还有个问题，引入标签时，他好像还会自动import

yes, just remove added import(s), same problem when using auto imported composables

**Comment by WangJincheng4869** [comment]
请问，您的 `tsconfig.json` 中有 `include` 对应的声明文件吗（比如 `auto-imports.d.ts` 和 `components.d.ts`），如果没有那么就是这个问题，因为我用的也是 webstorm，当然我用的是最新版本，因为websorm的bug真的很多，每次更新都会有不少的优化。

**Comment by zwmmm** [comment]
这个问题解决了吗 最新版本的 webstorm 2024.2 还是会有这个问题

**Comment by IF-tiger** [comment]
> 这个问题解决了吗 最新版本的 webstorm 2024.2 还是会有这个问题

在最新的eap版本中索引正常了
